src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -2940,6 +2940,20 @@ public override BoundNode VisitComplexConditionalReceiver(BoundComplexConditiona
             return null;
         }
 
+        public override BoundNode VisitComplexReceiver(BoundComplexReceiver node)
```

Comment:
Are any flow passes actually going to see one of these? It looks like they're only created during local lowering, so I wouldn't expected them to be seen.

Reply:
> Are any flow passes actually going to see one of these? It looks like they're only created during local lowering, so I wouldn't expected them to be seen.

This code path is reachable. Flow analysis is done during async rewrite, etc.